README.md

```diff
@@ -103,6 +103,12 @@ class UrlGenerator < Alephant::Broker::LoadStrategy::HTTP::URL
 end
 ```
 
+##### Revalidate Strategy
+
+```ruby
```

Comment:
Yup, already got that drawn out - was just going to ask you where to put it... replace the current NewsBeat one you originally linked me to?